deploy/build_rpanion.sh

```diff
@@ -8,8 +8,17 @@ cd ../
 # If less than 500Mb RAM, need to tell NodeJS to reduce memory usage during build
 if [ $(free -m | awk '/^Mem:/{print $2}') -le 500 ]; then
     export NODE_OPTIONS="--max-old-space-size=256"
+
+    # Also increase swap size to 1024MB if it is currently less than 1000MB
```

Comment:
Two things:
- This is RPi-specific, so needs to go in to ``RasPi2-3-4-5-deploy.sh``
- Does this work without a reboot? If the user needs to reboot to apply the changes, this should instead fail with an error telling the user to increase swap space and reboot their system.

Reply:
Moved it to the Pi2-3-4-5 file.

This works without a reboot. `dphys-swapfile swapoff` disables the swapfile and de-allocates the data from the swapfile, `dphys-swapfile setup` resizes the swapfile, and `dphys-swapfile swapon` re-enables it.

The `swapoff` command will fail if there isn't enough free memory to hold the initial contents of the swapfile, which ensures data won't be lost.